openpype/hosts/fusion/plugins/publish/collect_instances.py

```diff
@@ -57,6 +57,18 @@ def process(self, instance):
             start_with_handle = comp_start
             end_with_handle = comp_end
 
+        frame = instance.data["creator_attributes"].get("frame")
+        # explicitly publishing only single frame
+        if frame:
```

Comment:
This will never work for frame 0?
```suggestion
        if frame is not None:
```
Did you mean this?

I'd actually also add a comment that this if statement is for the `image` family just so it's clear "WHY" it's there.